### PR TITLE
update changelog and version

### PR DESCRIPTION
**Pull request by Blackbaud-TimPepper**
If some writers could give feedback on the changelog text, that would be helpful!

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,13 @@
+# 2.14.0 (2018-11-29)
+- Added `stache-hide-from-search` component to allow users to remove content from Stache Search results. [#557](https://github.com/blackbaud/stache2/pull/557)
```
Comment (blackbaud-johnly) [inline CHANGELOG.md]: In the Component docs, we don't format component names as code snippets. It probably doesn't matter here in the changelog, but if "Added hide from search component..." works just as well, it would be more consistent with the formatting in the Component docs.

Not sure if it matters, but should "remove" be "exclude"? I mean, are we excluding the content from the search altogether, or are we including it in the search and then removing it from the search results? I assume the former. If so, "exclude" might be the better word choice here.

Also, "search" at the end should not be capitalized.

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,3 +1,13 @@
...
+- Fixed alignment issue with sidebar and table of contents alignment. [#556](https://github.com/blackbaud/stache2/pull/556)
```
Comment (blackbaud-johnly) [inline CHANGELOG.md]: We don't need both instances of "alignment." I'd delete the second one, but you can delete the first instead if you think that reads better.

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,3 +1,13 @@
...
+- Updated version of `skyux-builder`.
```
Comment (blackbaud-johnly) [inline CHANGELOG.md]: Elsewhere in the changelog, we just say "SKY UX Builder" with that capitalization and without the code snippet formatting. I think we should use that style here as well unless there is a reason to format differently in this instance. 

Also, should there be a link to a PR here?

File: CHANGELOG.md [hunk 4]
```diff
@@ -1,3 +1,13 @@
...
+- Updated Travis build options to enhance test utility.
```
Comment (blackbaud-johnly) [inline CHANGELOG.md]: Should there be a link to a PR here?

File: CHANGELOG.md [hunk 5]
```diff
@@ -1,3 +1,13 @@
...
+- Updated skyux dependencies to depend on components directly. [#553](https://github.com/blackbaud/stache2/pull/553)
```
Comment (blackbaud-johnly) [inline CHANGELOG.md]: "skyux" should be "SKY UX"

FWIW, "dependencies to depend on" seems awkward. And I'm not sure what we mean by it.

File: CHANGELOG.md (hunk 1 above)
Comment (blackbaud-johnly) [reply CHANGELOG.md]: Actually, disregard the first point about the code snippet. It's not how we format component names in the docs, but it's consistent with the rest of the changelog. We might want to get those on the same page at some point, but we should be consistent within the changelog for now.

File: CHANGELOG.md [hunk 6]
```diff
@@ -1,3 +1,13 @@
...
+- Updated to allow `Action Button` comopnent to be used without icons.
```
Comment (blackbaud-johnly) [inline CHANGELOG.md]: Should there be a link to a PR here? I assume we should combine this bullet with the next bullet.

The placement of "to allow" is awkward and hard to follow. "Action Button" should not be capitalized; it should be the element name, which I think should be "sky-action-button" here. Also, "component" is misspelled. ... I'd combine this item with the next one as:

"Updated the `sky-action-button` component to improve accessibility and allow action buttons without icons."

File: CHANGELOG.md [hunk 7]
```diff
@@ -1,3 +1,13 @@
...
+- Updated `Action Button` component to be accessible. [#548](https://github.com/blackbaud/stache2/pull/548)
```
Comment (blackbaud-johnly) [inline CHANGELOG.md]: See previous comment.

File: CHANGELOG.md [hunk 8]
```diff
@@ -1,3 +1,13 @@
...
+- Update `Code Block` component to use accessible code highlighting coloring. [#551](https://github.com/blackbaud/stache2/pull/551)
```
Comment (blackbaud-johnly) [inline CHANGELOG.md]: "Updated" should be past tense like the other items. "Code Block" should not be capitalized; it should be the element name, which should be "stache-code-block" here. Also, "accessible code highlighting coloring" is a mouthful and hard to follow. Based on the PR description, I'd change this entry to "Updated `stache-code-block` component to improve accessibility." If you think we need to say more about the specific accessibility change, I'd "Updated `stache-code-block` component to improve accessibility by increasing the color contrast."

File: CHANGELOG.md (hunk 3 above)
Comment (Blackbaud-TimPepper) [reply CHANGELOG.md]: The ones that don't have a direct link to a PR are all included in the following entries. There's two instances in this update where one PR fixed several things.

Comment (Blackbaud-TimPepper) [reply CHANGELOG.md]: Changed the formatting to clarify this
